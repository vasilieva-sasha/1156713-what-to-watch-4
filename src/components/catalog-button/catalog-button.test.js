import React from "react";
import renderer from "react-test-renderer";
import CatalogButton from "./catalog-button";

describe(`CatalogButtonComponent`, () => {
  it(`button renders correctly`, () => {
    const tree = renderer.create(
        <CatalogButton onCatalogButtonClick={() => {}} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
