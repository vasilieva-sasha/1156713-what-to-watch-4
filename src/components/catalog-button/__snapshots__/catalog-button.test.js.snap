// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CatalogButtonComponent button renders correctly 1`] = `
<div
  className="catalog__more"
>
  <button
    className="catalog__button"
    onClick={[Function]}
    type="button"
  >
    Show more
  </button>
</div>
`;
